modules/gitops-release-notes-1-11-0.adoc

```diff
@@ -0,0 +1,39 @@
+// Module included in the following assembly:
+//
+// * release_notes/gitops-release-notes.adoc
+
+:_content-type: REFERENCE
+[id="gitops-release-notes-1-11-0_{context}"]
+= Release notes for {gitops-title} 1.11.0
+
+{gitops-title} 1.11.0 is now available on {OCP} 4.12, 4.13, and 4.14.
+
+[id="errata-updates-1-11-0_{context}"]
+== Errata updates
+
+[id="gitops-1-11-0-security-update-advisory_{context}"]
+=== RHSA-2023:NNNN - {gitops-title} 1.11.0 security update advisory
+
+Issued: 2023-12-07
+
+The list of security fixes that are included in this release is documented in the following advisory:
+
+* link:https://access.redhat.com/errata/RHSA-2023:NNNN[RHSA-2023:NNNN]
+
+If you have installed the {gitops-title} Operator in the default namespace, to view the container images in this release, run the following command:
+
+[source,terminal]
+----
+$ oc describe deployment gitops-operator-controller-manager -n openshift-gitops-operator
+----
```

Comment:
@masauso-lungu Do we have any security advidory this release? If not we can remove this section.

Please clarify with the SMEs. Thank you!

Reply:
It's just a regular enhancement advisory this time!

Reply:
The available errata is an enhancemnt and i was due to update it. @reginapizza, do we need this RHEA published as we normally do with RHSA?, thank you

Reply:
I'm not sure of the question @masauso-lungu... would this not be a question for @Srivaralakshmi ?

Reply:
@masauso-lungu AFAIU, we only have a regular enhancement advisory this time! If there is no security advisory, then please drop this section. I hope this answer clarifies the confusion.

Thanks, @reginapizza for the response. This is helpful.

Reply:
Thank you @Srivaralakshmi . I will drop it.